app/modules.go

```diff
@@ -44,17 +43,20 @@ func (BankModule) DefaultGenesis(cdc codec.JSONCodec) json.RawMessage {
 				Denom:    appparams.BondDenom,
 				Exponent: 0,
 				Aliases: []string{
-					"microumee",
+					"microumee", appparams.BaseExtraDenom,
```

Comment:
Since we're already been running with an alias, I don't have any more concerns about adding another one here